### PR TITLE
increased user menu font size and removed bold

### LINKED ISSUE #471: Increase font size on .action-menu.user-menu .button
currently the user button in the header inherts a 0.85em font size. I'm thinking this should probably be overridden to be 1em to match the mock and to provide better usability/visual balance.

@jenzweben can you review?

Ref #469, #188

### PR DESCRIPTION
**Pull request by jenzweben**
Fixes #471

### CODE REVIEW

File: app/styles/components/_action-menu.scss [hunk 1]
```diff
@@ -43,8 +43,9 @@ $dropdown-distance-from-menu: $dropdown-padding * 2;
     }
   }
   &.user-menu .button {
-    font-size: $base-font-size * 0.85;
-    color: white;
+//    font-size: $base-font-size * 0.85;
```
Comment (jrjohnson) [inline app/styles/components/_action-menu.scss]: please remove this commented code.

**Comment by saschaben** [comment]
visual check is a PASS.

**Comment by jrjohnson** [comment]
@jenzweben we can merge this with failing tests you don't need to go through the rebase dance, but I have the one change listed above.  Let me know if you want some help with that.

**Comment by jenzweben** [comment]
OK I committed and pushed the fixed file...lmk if I did it incorrectly

**Pull request merged**